docs/oauth.md

```diff
@@ -118,7 +119,6 @@ This set of OAuth Config capabilities are not shared with the regular connector
 | OAUTH_CAP_REQUIRE_PKCE | Whether your OAuth provider supports PKCE, more detials: https://oauth.net/2/pkce/ | false | true |
 | OAUTH_CAP_PKCE_REQUIRES_CODE_CHALLENGE_METHOD | Whether your OAuth provider PKCE requires code_challenging_method passed in. If set to true, we are using S256 by default. | false | true |
 | OAUTH_CAP_SUPPORTS_STATE | Used to protect against CSRF attacks, more details: https://auth0.com/docs/protocols/state-parameters | false | true |
-| OAUTH_CAP_REQUIRES_VERIFY_STATE | Used together with OAUTH_CAP_SUPPORTS_STATE | false | true |
```

Comment:
Did you mean to remove this capability?

Reply:
Yeah this is removed as of 21.3+ version, we'll only need OAUTH_CAP_SUPPORTS_STATE